admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -73,6 +73,15 @@ Traefik
 
 Thanks to `@pauvos <https://github.com/pauvos>`_ for traefik example.
 
+HAProxy
+^^^^^^^
+::
+
+  acl dav path /.well-known/carddav /.well-known/caldav
```

Comment:
Thanks :+1: I'm not familiar with ha proxy. Could you explain what the code does?